app/views/mobileValidate.scala.html

```diff
@@ -41,7 +41,12 @@
             <div id="svv-panorama-outline">
                 <div id="svv-panorama">
                     <div id="view-control-layer-mobile" style="z-index: 2">
-                        <div id="label-description-box"></div>
+                        @if("Amsterdam" == cityName){
+                            <div id="label-description-box" class="label-description-box amsterdam"></div>
+                        }
+                        @if("Amsterdam" != cityName){
```

Comment:
You should be able to use `else` here instead of another `if`. If you were having trouble getting it to work, try playing with the whitespace around the parentheses or curly braces. I think that that can sometimes act in weird ways in the `.scala.html` files.